### ISSUE TITLE
Importing into Power Plattform Environment without Dynamics Database

### ISSUE THREAD

**Issue by sbetzin** [demianrasko/Dynamics-365-Workflow-Tools#236]
Hi There, 
I am using this great toolset in a new environment (2021 release wave 1 - 9.2.21023.00135) and the importing fails with error:

Plugin Assemblies import: FAILURE. Error: Plugin: msdyncrmWorkflowTools, Version=1.0.61.1, Culture=neutral, PublicKeyToken=416e876b9bee261e of PluginTypeName: Microsoft.Crm.Entities.PluginType and PluginTypeNode: caused an exception.: Plugin Types import: FAILURE. Error: Plugin: msdyncrmWorkflowTools, Version=1.0.61.1, Culture=neutral, PublicKeyToken=416e876b9bee261e of PluginTypeName: Microsoft.Crm.Entities.PluginType caused an exception.: Invalid CrmReferenceTarget: The entity list doesn't exist. Property: Marketing

I suppose this is the case because I am using a clean Power Plattform environment without a dynamics sales application.
It would be great to be "entity neutral" to use this tool in other environments as well.

Thanks!

Sebastian

**Comment by AnaChivite** [comment]
Hello!

I have the same problem.
Would be possible to have an "entity neutral" version ?

Thanks!
Ana

**Comment by james-tekstack** [comment]
I agree. We needed an entity neutral version, so I created one. My fork has a new project that excludes classes with dependencies on Microsoft Sales and Customer Service. I've created a release that deploys into a stock Power Platform environment. 

https://github.com/james-tekstack/Dynamics-365-Workflow-Tools/releases/tag/1.0.61.1

I also created a pull request to merge these changes into the master branch. Hopefully @demianrasko will approve this.
